site/netlify.toml

```diff
@@ -12,3 +12,8 @@
   for = "/favicon.ico"
   [headers.values]
   cache-control = "max-age=604800" # 1 week
+
+[[redirects]]
+from = "/guide/inline-queries.html"
+to = "/plugins/inline-queries.html"
```

Comment:
```suggestion
to = "/plugins/inline-query.html"
```

Reply:
Similar redirects need to be set up for all other languages.

Reply:
> to = "/plugins/inline-query.html"

But the new file name is `inline-queries.md`

> Similar redirects need to be set up for all other languages.

Maybe let the translator do them by their self. Just in case if this PQ is merged before the translation progress is completed.

Reply:
> > to = "/plugins/inline-query.html"
> 
> But the new file name is `inline-queries.md`

Good catch, it should be renamed for consistency with the other built-in plugins.

> > Similar redirects need to be set up for all other languages.
> 
> Maybe let the translator do them by their self. Just in case if this PQ is merged before the translation progress is completed.

Alright.